examples/src/main/scala/examples/hybrid/HybridApp.scala

```diff
@@ -58,11 +58,6 @@ class HybridApp(val settingsFilename: String) extends Application {
                                                      (networkControllerRef, nodeViewHolderRef, localInterface,
                                                       HybridSyncInfoMessageSpec, settings.network, timeProvider))
 
-  //touching lazy vals
```

Comment:
why these lines are removed?

Reply:
Because the `val`s are not `lazy val`s anymore. So, there is no point in touching them. They are already touched when they are created.